### PR TITLE
Post Title

### LINKED ISSUE #139: Add a title field to the post
The spec requires each post to have a title and a body and right now we only have a body.

### PR DESCRIPTION
**Pull request by maharshmellow**
Fixes #139 

Adds a post title since the requirements mention that each post also has a title.

### CODE REVIEW

File: src/components/post/Post.jsx [hunk 1]
```diff
@@ -169,6 +169,7 @@ class Post extends Component {
       <div className="post-block">
         {this.renderMenu()}
         { post.imageSrc ? <img className="post-img" src={post.imageSrc} alt="more-icon" /> : null }
+        <h5 className="post-title"><em>{post.title}</em></h5>
```
Comment (ceciliaccwei) [inline src/components/post/Post.jsx:174]: Is it possible to decrease the space between the username and the title? It seems to be a lot larger than before. Might be caused by the h5 margin?

**Review by ceciliaccwei — CHANGES_REQUESTED** [review]
Checked out locally and reviewed the code!
Overall perfect😄 But two things I would suggest:
1. decrease the space between the username and the post title a little bit since it seems to be a lot larger than before.
2. Consider italicize the title in the editable post component as well? Since the title is italic everywhere else. (Whats your idea on this?)

File: src/components/post/Post.jsx (hunk 1 above)
Comment (maharshmellow) [reply src/components/post/Post.jsx:174]: done.. also fixed an issue where there was no spacing at the top of the post preview dialog.

**Review by ceciliaccwei — APPROVED** [review]
Looks good😄

**Pull request merged**